COMPONENTS.md

```diff
@@ -0,0 +1,160 @@
+# Form Components
+
+A set of form components provides support the the styled form elements in the CD Framework. 
+
+## Form
+
+The form component creates the required fieldset and legend.
+
+Use wire:submit on the form component rather than wire:click on the submit button to maintain the 
+expected behavior of submitting the form if the user hits enter in a text element.
+
+Here is an example of a simple form:
+
+```
+<x-cd.form.form legend="Simple form" wire:submit="submit">
+  <x-cd.form.text label="Name" wire:model="name"/>
+  <div>
+    <x-cd.form.submitbutton/>
+    <x-cd.form.resetbutton/>
+  </div>
+</x-cd.form.form>
```

Comment:
I wonder if we should move "form" up one level so that this can be `x-cd.form` rather than `x-cd.form.form`. Since we are in the "components/cd" directory anyway, that seems mostly good to me.

There's also the option to use [anonymous index components](https://laravel.com/docs/10.x/blade#anonymous-index-components). I find it a little indirect, though, since `x-cd.form` would be located at "components/cd/form/index.blade.php".

I think it would be worth discussing to see if there's a group preference.

Reply:
@woodsowl I am inclined to leave the form element in the form directory, but glad to hear more views.

Reply:
moved as suggested